src/vs/workbench/contrib/customEditor/browser/commands.ts

```diff
@@ -172,3 +172,54 @@ MenuRegistry.appendMenuItem(MenuId.CommandPalette, {
 		model.redo();
 	}
 }).register();
+
+(new class ToggleCustomEditorCommand extends Command {
+	public static readonly ID = 'editor.action.customEditor.toggle';
+
+	constructor() {
+		super({
+			id: ToggleCustomEditorCommand.ID,
+			precondition: CONTEXT_HAS_CUSTOM_EDITORS,
+			kbOpts: {
+				primary: KeyMod.CtrlCmd | KeyCode.KEY_E,
```

Comment:
Let's omit the keybinding for now. We typical only have default keybindings for more common editor actions. Users could configure a keybinding for this command if they wish